### DISCUSSION TITLE
TANDEM X20S, TD SR10 can not get Stabilization to work.

### DISCUSSION THREAD

**Discussion by dcepello** [FrSkyRC/ETHOS-Feedback-Community#4311] [General]
FW-TD-ISMR_v2.2.6
Ethos-suite-Windows -1.8.5
FW-TD SR10_v1.0.11
LUA FrSky-STAB_RX_v2.0.6
 
I am unable to get the stabilization to work on my receiver. I have gone through the You tube videos time and time again and still don't understand what I am doing wrong. Your help would be most welcome.+

**Comment by Chilie78** [comment]
Can you post a screenshot of your mixing page? In order to narrow down what the problem could be, I would need to know what channels you are using. Did you set it up as per the manual? Have you tried the new v3 lua scripts?

**Comment by dcepello** [comment]
I am new at frsky,  so I Am using basic stabiization.

Sent from my T-Mobile 5G Tablet
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Chilie78** [comment]
That doesn't answer my question. Selecting basic stabilization is only the first part of what you need to do. The manual talks about what channels need to be plugged into which ports. You need to set the gain and an ability to turn off the stabilization which is why I asked to see your mixer page to see if you've skipped this important step.
